packages/clarity-visualize/src/layout.ts

```diff
@@ -150,34 +141,14 @@ export class LayoutHelper {
             return;
         }
 
-        this.styleSheetMap[documentId] = styleIds;
```

Comment:
Has the previous method caused problems, or is this just part of the ongoing adoptedStyleSheets refactoring effort??

Reply:
Yeah the method I had here of creating tags was really expensive and making the DOM rendering take forever. I had built this whole process before because I couldn't share style sheets across documents - but now (on line 117) I found out how to create a style sheet within the iframe.